recipes/preprocess/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.2.2" %}
+{% set git_rev = "d5ab9a0544cd2b15b0e8ae4f0b143b9047d1a54b" %}
+
+package:
+  name: preprocess
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hplgit/preprocess.git
+  git_rev: {{ git_rev }}
```

Comment:
Can we please download a tarball instead? Maybe need to ask upstream to add some tags if we go this route.